### DISCUSSION TITLE
Series within Collection tab

### DISCUSSION THREAD

**Discussion by PleaseStopAsking** [LeoKlaus/plappa#13] [Feature Requests]
With support for both series and collections, it would be nice to be able to see all series together. Since there is already a collections tab, it could be used to also showcase series alongside any collections without having to use the Books tab or creating another tab altogether.

Just an idea

**Comment by LeoKlaus** [comment]
Thanks for the suggestion! I don't know about mixing series with collections, but I agree there should be a list. My current plan is to allow users to modify the tabs and their order.

**Reply by PleaseStopAsking** [reply]
The idea of allowing users to define which tabs they see sounds awesome and would solve the series tab issue as well. Thanks.

**Comment by mandopatriot** [comment]
How do we currently view Series in the app? I'm at the latest version 1.1.0, but only see Collections, which doesn't have anything because I didn't create any yet.

**Reply by LeoKlaus** [reply]
Series are visible in the "Books" tab. By default, all books will be shown, but you can choose to collapse series in the top right menu:
![Screenshot 2024-04-04 at 18 07 04](https://github.com/LeoKlaus/plappa/assets/66492595/bec7b95c-a25f-41c0-ab71-9af591e39aa4)

This will show series as a single item (see Harry Potter in my screenshot).

**Reply by mandopatriot** [reply]
Thank you! Did not notice that. It might make more sense to have Series listed as a separate tab or included in Collections as noted in this thread.

**Comment by insanelysick** [comment]
Definitely would welcome the addition of a series tab. Perhaps even a "next up" section on the home tab that shows the next book in a series so you can jump right in.

**Comment by LeoKlaus** [comment]
Closed with 1.1.4
